perun-rpc/src/main/perl/Perun/beans/User.pm

```diff
@@ -177,11 +222,11 @@ sub getSortingName {
 
 sub getCommonArrayRepresentation {
 	my $user = shift;
-	return ($user->getId, $user->getDisplayName);
+	return ($user->getId, $user->getDisplayName, $user->getIsServiceUser, $user->getIsSponsoredUser);
```

Comment:
In order to get nice print of booleans you can use implementation used in Attribute object, method getValueAsScalar(). 
```perl
($value) ? 'true' : 'false';
```
You can also check, that value is actually JSO boolean 
```perl
if (ref $value eq "JSON::PP::Boolean")
```
If you would need to set it back (since you can't pass boolean from terminal console) then you can use: 
```perl
my $true = JSON::PP::true;
my $false = JSON::PP::false;
```
and pass it to setter method after checking input for 1/0 or 'true'/'false'. Undef also means 'false'.